### ISSUE TITLE
Define target format for "dita.parser" extension

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#3296]
I have this usecase in which a JSON file is referenced in the DITA Map and using the "dita.parser" extension I could try to dynamically convert it to a DITA Map having keydef's for each JSON key/value pair. But the custom parsed content needs to be "seen" as a DITA Map, not as a DITA topic.

**Comment by jelovirt** [comment]
This would require that we deduce from the file name that the output will be a map. My preference is to not use file names to choose the parser.

**Comment by raducoravu** [comment]
Agreed. How about if dita parser implementation could implement additional interface through which it could provide the format of the parsed content?

**Comment by raducoravu** [comment]
I encountered the same need, we wanted to dynamically convert Word to DITA and instead of generating a large DITA topic I wanted to generate a DITA Map.
Maybe add a "ditamap.parser" extension?
